WcaOnRails/config/routes.rb

```diff
@@ -87,6 +87,8 @@
 
   get 'robots' => 'static_pages#robots'
   get 'server-status' => 'server_status#index'
+  get 'edit_translation' => 'git_hub#edit_translation', as: :edit_translation
+  patch 'update_translation' => 'git_hub#update_translation', as: :update_translation
```

Comment:
nit: I have a slight preference for `-` in urls instead of `_`. I realize we're inconsistent about this currently, but let's go `-` from here out.